examples/decoding/plot_receptive_field.py

```diff
@@ -0,0 +1,157 @@
+mne.viz.tight_layout()
+
+###############################################################################
+# Create and fit a receptive field model
+# --------------------------------------
+#
+# We will construct a model to find the linear relationship between the EEG
+# signal and a time-delayed version of the speech envelope. This allows
+# us to make predictions about the response to new stimuli.
+
+# Define the delays that we will use in the receptive field
+tmin, tmax = -.4, .2
+
+# Initialize the model
+spec_names = ['freq_%s' % ii for ii in range(speech.shape[0])]
+rf = ReceptiveField(tmin, tmax, sfreq,
+                    feature_names=spec_names, model=Ridge(alpha=1.))
+n_delays = int((tmax - tmin) * sfreq) + 2  # +2 to account for 0 + end
+
+n_splits = 3
+cv = KFold(n_splits)
+
+# Prepare model data
+Y, _ = raw[:]  # Outputs for the model
+Y = np.rollaxis(Y, -1, 0)  # Make time the first dimension
+speech = np.rollaxis(speech, -1, 0)
+
+# Iterate through folds, fit the model, and predict/test on held-out data
+coefs = np.zeros([n_splits, n_channels, n_delays])
+scores = np.zeros([n_splits, n_channels])
+for ii, (train, test) in enumerate(cv.split(speech)):
+    print('CV iteration %s' % ii)
+    rf.fit(speech[train], Y[train])
+    preds = rf.predict(speech[test])
+
+    # mask_pred tells us which points were removed / kept in time delaying
+    for jj, i_ch in enumerate(Y.T):
+        scores[ii, jj] = pearsonr(Y[:, jj][test][rf.mask_predict_],
+                                  preds[:, jj])[0]
+    # Remove features dimension because there's only one item in it
+    coefs[ii] = rf.coef_
+delays_time = rf.delays_ / float(sfreq)
+# Average scores and coefficients across CV splits
+mn_coefs = coefs.mean(0)
+mn_scores = scores.mean(0)
+
+# Plot mean prediction scores across all channels
+fig, ax = plt.subplots()
+ix_chs = np.arange(n_channels)
+ax.plot(ix_chs, mn_scores)
+ax.axhline(0, ls='--', color='r')
+ax.set(title="Mean prediction score", xlabel="Channel", ylabel="Score ($r$)")
+mne.viz.tight_layout()
+
+###############################################################################
+# Investigate model coefficients
+# ------------------------------
+#
+# Finally, we will look at how the linear coefficients (sometimes
+# referred to as beta values) are distributed across time delays as well as
+# across the scalp. We will recreate `figure 1`_ and `figure 2`_ from [1]_.
+
+# Print mean coefficients across all time delays / channels (see Fig 1 in [1])
+time_plot = -.180  # For highlighting a specific time.
+fig, ax = plt.subplots(figsize=(4, 8))
+vmax = np.abs(mn_coefs.max())
+vmin = -vmax
+ax.pcolormesh(delays_time, ix_chs, mn_coefs, cmap='RdBu_r',
+              vmin=vmin, vmax=vmax)
+ax.axvline(time_plot, ls='--', color='k', lw=2)
+ax.set(xlabel='Lag (s)', ylabel='Channel', title="Mean Model\nCoefficients",
+       xlim=[delays_time.min(), delays_time.max()], ylim=[0, len(ix_chs)])
```

Comment:
ok cool, let's see how the latest push looks. Sorry that I keep pushing upon every little doc update, btw. I still haven't been able to get the docs building on my computer since I upgraded to python 3.5. I can't figure out how to get VTK (and thus Mayavi) installed...